salt/_modules/caasp_pillar.py

```diff
@@ -26,3 +26,16 @@ def get(name, default=''):
         elif res.lower() in ["false", "no", "off"]:
             return False
     return res
+
+
+def get_feature_gates_flags():
```

Comment:
Having this function only for the _feature_gates_ does not look right to me. I would generalize it, something like:

```Python
def get_map(name, separator):
    return ",".join(['{0}{2}{3}'.format(key, separator, fg[key]) for key in get(name, {})])
```

Reply:
Either generalize it here, or have another module `caasp_kubernetes` with specific kubernetes slang.